feature/header - Add header in my-library

SylviaStachera opened the pull request:
Add header by js

SylviaStachera committed:
feature/header - Add header in my-library

SylviaStachera committed:
Merge branch 'main' of https://github.com/Okazaki92/Quattro-Team-project-filmoteka into feature/header

mkubera commented on src/js/headerLibrary.js:
```diff
@@ -0,0 +1,50 @@
+  e.preventDefault();
+
+  headerHome.classList.add('header-library');
+
```
del line

mkubera commented on src/js/headerLibrary.js:
```diff
@@ -0,0 +1,50 @@
+
+  //===========change active btn===============
+
+  btnQueue.addEventListener('click', e => {
+    e.preventDefault();
+
+    btnWatched.classList.remove('btn-current-active');
+    btnQueue.classList.add('btn-current-active');
+  });
+
+  btnWatched.addEventListener('click', e => {
+    e.preventDefault();
+
+    btnQueue.classList.remove('btn-current-active');
+    btnWatched.classList.add('btn-current-active');
+  });
```
- kazdy listener dostaje cb fn
- `toggleBtnActive`, tkora uzywa `classList.toggle` dla obydwu btnow
- `e.preventDefault();` jest zbedne (dos prawdzenia)

mkubera reviewed (changes requested):
do upiekszenia :-]

SylviaStachera committed:
Add toggleBtnActive to both btn in header library

Okazaki92 reviewed (approved):
Super, dzięki 😁

SylviaStachera committed:
Add pagination

Change:
index.html - import pagination.html,
index.js - import pagination.js,
getGallery.js - change "getTrendingGallery"
search.js - change :#form__search -> #formSearch
import pagination.scss to main.scss

Okazaki92 committed:
Merge branch 'main' into feature/header
